app/poros/fuel_station.rb

```diff
@@ -0,0 +1,23 @@
+class FuelStation
+  attr_reader :name,
+              :access_times,
+              :street_address,
+              :city,
+              :state,
+              :zip,
+              :concat_address,
+              :fuel_type,
+              :distance_miles
+
+  def initialize(data)
+    @name = data[:fuel_stations][0][:station_name]
+    @access_times = data[:fuel_stations][0][:access_days_time]
+    @street_address = data[:fuel_stations][0][:street_address]
+    @city = data[:fuel_stations][0][:city]
+    @state = data[:fuel_stations][0][:state]
+    @zip = data[:fuel_stations][0][:zip]
+    @fuel_type = data[:fuel_stations][0][:fuel_type_code]
+    @distance_miles = data[:fuel_stations][0][:distance]
+    @concat_address = @street_address + ', ' + @city + ', ' + @state + ' ' + @zip
```

Comment:
This works. You could also create a helper method that does the concatting and then call that method in your initialize. Could look like:
```
def concat_address
    @street_address + ', ' + @city + ', ' + @state + ' ' 
    + @zip
end
```
Then in your initialize you could 
```
@concat_address = concat_address
```